nixos/modules/services/networking/nix-serve.nix

```diff
@@ -31,6 +31,9 @@ in
         default = null;
         description = ''
           The path to the file used for signing derivation data.
+          Generate with: nix-store --generate-binary-cache-key cache.example.com-1 nix-serve.sec nix-serve.pub
```

Comment:
Can we use the same example as in the Nix and nix-store manuals for consistency? Perhaps even refer to the man page as well.

```nix-store --generate-binary-cache-key key-name secret-key-file public-key-file```

Reply:
Hi Sarah! :-) Thanks for the review - fixed! - I am not sure what format stuff is supported in the description, but from reading other options I figured basic markdown should be available?